source/documentation/disaster_recovery_guide/task/task_ConfigureStretchCluster.adoc

```diff
@@ -42,7 +42,7 @@ The standalone {engine-name} is only highly available when managed externally. F
 * In a public cloud.
 ====
 
-*Configuring the Standalone {engine-name} Stretch Cluster*
+.Procedure
```

Comment:
IMHO too generic word, unless that's the common practice in RH documentation.

Reply:
It's standard practice at Red Hat. It's prescribed in the Modular Documentation Reference Guide.